### PR TITLE
bugfix/01-Overlay-ScrollLock

### PR DESCRIPTION
**Pull request by Sanjay-B**
Internal bug report via [issue 59](https://github.com/deviap/deviap-main/issues/59) filed internally by a @deviap-dev (Jason). 

Aimed to fix: 
https://github.com/deviap/deviap-main/issues/59

Other additions:
- Make scrollbar in collectionLayoutAuto invisible when enabled.
- Fixed https://github.com/deviap/deviap-main/issues/78

### CODE REVIEW

File: source/libraries/UI/constraints/controllers/collectionLayoutAuto.lua [hunk 1]
```diff
@@ -1,6 +1,7 @@
 return function(params)
 	local public = {}
 	public.container = core.construct("guiScrollView", params or {})
+	public.container.scrollbarAlpha = 0
```
Comment (u-train) [inline source/libraries/UI/constraints/controllers/collectionLayoutAuto.lua:4]: This breaks output. Use the params table or manually set it _outside_ of collectionLayoutAuto function.

Comment (u-train) [reply source/libraries/UI/constraints/controllers/collectionLayoutAuto.lua:4]: imo, this is unexpected behavior, another reason this shouldn't be a thing.

Comment (Sanjay-B) [reply source/libraries/UI/constraints/controllers/collectionLayoutAuto.lua]: > This breaks output. Use the params table or manually set it _outside_ of collectionLayoutAuto function.

Addressed / corrected in in https://github.com/deviap/deviap-main/pull/77/commits/07e40ec4142c6d9842b247da65bd1b35d4c50f5a  & https://github.com/deviap/deviap-main/pull/77/commits/0212889a78b68967e6c311ad7821ee0f70c41ef4 .

**Comment by u-train** [comment]
This looks good, absolutely hot, and works on my side. Merge OK!

**Review by u-train — APPROVED** [review]
This looks good, absolutely hot, and works on my side. Merge OK!

**Pull request merged**